pages/use-solid.md

```diff
@@ -8,43 +8,51 @@ redirect_from:
-| [solidweb.org](https://solidweb.org) | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de) | Germany |
 
-If you are a provider you can add your service to this list by emailing [info@solidproject.org](mailto:info@solidproject.org).
+| Provider                                           | Responsible for Domain Name and Terms                                      | Responsible for Hosting                       | Hosting Location |
+|----------------------------------------------------|----------------------------------------------------------------------------|-----------------------------------------------|------------------|
+| [Inrupt Pod Spaces](https://signup.pod.inrupt.com) | [Inrupt, Inc.](https://inrupt.com/terms-of-service)                        | [Amazon](https://aws.amazon.com)              | USA              |
+| [inrupt.net](https://inrupt.net)                   | [Inrupt, Inc.](https://inrupt.com/terms-of-service)                        | [Amazon](https://aws.amazon.com)              | USA              |
+| [solidcommunity.net](https://solidcommunity.net/)  | [Solid Project](https://github.com/solid/solidcommunity.net_operations)    | [Digital Ocean](https://www.digitalocean.com) | UK               |
+| [solidweb.org](https://solidweb.org)               | [Solid Grassroots](https://gitlab.com/groups/solidweb.org/-/group_members) | [Hosteurope](https://www.hosteurope.de)       | Germany          |
 
-## How to self-host your Pod
+If you are a provider who wants to add your service to this list, email [info@solidproject.org](mailto:info@solidproject.org).
 
-The way to get most control of your data is to **self-host**. Self-hosting means that your data is physically stored on a hard drive you hold at home, and managed by [a software](/for-developers/pod-server) you run on your own machine. This way, you don't have to entrust any third party with your data. Currently, this option still requires some technical background, and it may not be suitable for everyone yet.
 
-We would like to make self-hosting a more user-friendly option in the future. The great thing about Solid is the flexibility it offers: you can choose to outsource the work for now and pick a provider, and **move to self-hosting later** on when the tools are ready for you.
+## Self-host Your Pod
+
+Self-hosting your own Pod (i.e., running your own [Pod Server](/for-developers/pod-server)) provides the most control of your data. This way, you don't have to entrust any third party with your data. 
+
+Self-hosting requires some technical overhead and may not be suitable for everyone yet.  In the future, we would like to make self-hosting a more user-friendly option. The great thing about Solid is the flexibility it offers: you can choose to get a Pod from a provider now, and **move to self-hosting later**.
+
+# Next Steps
+
+When you get a Pod, you are also issued a unique WebID. 
```

Comment:
Heh ... forgot to add another sentence here to explain WebID and tie with the next paragraph. Ugh.  Will put up a patch sometime tomorrow.

Reply:
Updated.